docs/guides/cloud-setup-invited.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: Set up your IBM Cloud account - invited users
+description: Setup instructions for IBM Quantum on IBM Cloud to submit Qiskit SDK and Qiskit Runtime jobs for users with email invitations
+platform: cloud
+---
+
+{/* cspell:ignore urlendcoded, WQVY */}
+
+# Set up your IBM Cloud account - invited users
+
+If you do not have an invitation from an administrator to join an account, follow the steps in [Set up your IBM Cloud account](/docs/guides/cloud-setup) instead.
+
+You can run workloads on IBM&reg; quantum processing units (QPUs) by setting up an account on IBM Cloud&reg;. You will also need to [install and set up Qiskit and Qiskit Runtime](/docs/guides/install-qiskit#local).
+
+
+Your user account is associated with one or more [instances](./instances) that give access to IBM Quantum services. Additionally, a unique token is assigned to each account -- known as a *Cloud Resource Name* (CRN) -- allowing for IBM Quantum access from Qiskit.  For instructions to choose a specific instance, see [Connect to an instance](./instances#connect-instance).
+
+    <Admonition type="note">
+      The [IBM Quantum Platform dashboard](https://quantum.cloud.ibm.com) and the [Instances page](https://quantum.cloud.ibm.com/instances) list the instances that you can access.
+    </Admonition>
+
+<span id="cloud"></span>
+## Set up to use IBM Cloud
+
+### Before you begin
+
+1. Ensure you are working in an active Python environment with the [Qiskit SDK and Qiskit Runtime installed](/docs/guides/install-qiskit#local).
+
+1. Accept the email invitation.
+1. If you do not already have one, set up an [IBM Cloud account.](https://quantum.cloud.ibm.com/registration)
+   <Admonition type="note">
+   You can have multiple IBM Cloud accounts.  You can access any of your IBM Cloud accounts at any time from the account switcher in the top header of the [IBM Quantum Platform interface](https://quantum.cloud.ibm.com).
+   </Admonition>
+
+<span id="access-credentials"></span>
+### 1. Find your access credentials
+    1. Find your API key. From the [dashboard](https://quantum.cloud.ibm.com/), create your API key, then copy it to a secure location so you can use it for authentication.
+    2. Optional: Find your CRN from the Instances section of the dashboard. Click the icon to copy your CRN for the instance you want to use, then save it in a secure location so you can use it to identify the instance.
```

Comment:
or list all the instances you have access to in the selected account in https://quantum.cloud.ibm.com/instances and find the CRN of the instance you want to use.